packages/evm/package.json

```diff
@@ -30,34 +31,19 @@
     "access": "public"
   },
   "dependencies": {
-    "@react-icons/all-files": "^4.1.0",
-    "@sei-js/core": "^3.1.1",
-    "buffer": "^6.0.3",
-    "process": "^0.11.10",
-    "styled-components": "^6.1.1"
-  },
-  "peerDependencies": {
-    "react": "^17.0.0 || ^18.0.0",
-    "react-dom": "^17.0.0 || ^18.0.0"
+    "ethers": "^6.11.1",
```

Comment:
i think these should be moved into peer dependencies as well bc users will likely be using one or the other not both